core/vm/logger.go

```diff
@@ -142,27 +141,32 @@ func (l *StructLogger) CaptureStart(from common.Address, to common.Address, crea
 
 // CaptureState logs a new structured log message and pushes it out to the environment
 //
-// CaptureState also tracks SSTORE ops to track dirty values.
+// CaptureState also tracks SLOAD/SSTORE ops to track storage change.
 func (l *StructLogger) CaptureState(env *EVM, pc uint64, op OpCode, gas, cost uint64, memory *Memory, stack *Stack, rStack *ReturnStack, contract *Contract, depth int, err error) error {
```

Comment:
At some point, we should also update the method signature to mimic the opcodes, using a CallContext instead of passing all these parameters...

Reply:
Yes, but `callCtx` is an inner structure. Is it ok to uppercase it?

Reply:
I didn't mean you have to do it in this PR, really. 
(but _I_ think we can uppercase it  if needed)